bin-src/init.ts

```diff
@@ -0,0 +1,142 @@
+
+export const TestFramework = {
+    STORYBOOK: 'storybook',
+    PLAYWRIGHT: 'playwright',
+    CYPRESS: 'cypress'
+};
+type TestFrameworkType = typeof TestFramework[keyof typeof TestFramework];
+
+export const getPackageManagerInstallCommand = async (args: string[]) => {
+    return await getCliCommand(parseNi, args, { programmatic: true });
+};
+
+const getSBConfigFilePath = async () => {
+    // Walks up directory tree to find nearest Storybook config file.
+    return await findUp(['.storybook/main.ts', '.storybook/main.js', '.storybook/main.tsx', '.storybook/main.jsx', '.storybook/main.mjs', '.storybook/main.cjs'])
+}
+
+export const addChromaticScriptToPackageJson = async ({ packageJson, packagePath }) => {
+    try {
+        const json = {
+            ...packageJson,
+            scripts: {
+                ...packageJson?.scripts,
+                chromatic: `npx chromatic@latest`
+            }
+        }
+        await writeFile(packagePath, json, { spaces: 2 });
+    } catch (e) {
+        console.warn(e)
+    }
+}
+
+export const createChromaticConfigFile = async ({configFile, buildScriptName = null}) => {
+    await writeFile(configFile, {
+        ...(buildScriptName && {
+            buildScriptName
+        }),
+        skip: "dependabot/**"
+    });
+}
+
+export const installArchiveDependencies = async (packageJson: PackageJson, testFramework: TestFrameworkType) => {
+    let installArgs = ['-D', 'chromatic',`chromatic-${testFramework}`, 'storybook@next', '@storybook/addon-essentials@next', '@storybook/server-webpack5@next']
+    const sbConfigPath = await getSBConfigFilePath()
+    const sbVersion = packageJson?.devDependencies?.storybook || packageJson?.dependencies?.storybook
+    if(sbConfigPath && sbVersion) {
+        installArgs = ['-D', 'chromatic',`chromatic-${testFramework}`, `@storybook/server-webpack5@${sbVersion}`]
+    }
+    const installCommand = await getPackageManagerInstallCommand(installArgs)
+    await execaCommand(installCommand)
+}
+
+const intializeChromatic = async ({testFramework, packageJson, packagePath}: {testFramework: TestFrameworkType, packageJson: PackageJson, packagePath: string }) => {
+    await addChromaticScriptToPackageJson({ packageJson, packagePath });
+    await createChromaticConfigFile({configFile: 'chromatic.config.json'})
+    switch (testFramework) {
+        case TestFramework.CYPRESS:
+            await installArchiveDependencies(packageJson, TestFramework.CYPRESS)
+            break;
+        case TestFramework.PLAYWRIGHT:
+            await installArchiveDependencies(packageJson, TestFramework.PLAYWRIGHT)
+            break;
+    
+        default:
+            break;
+    }
+}
+
+export async function main(argv: string[]) {
+    const { flags } = meow(
+        `
+        Usage
+          $ chromatic init [-f|--framework]
+
+        Options
+          --framework, -f <framework     Test Framework that you are aiming to use with Chromatic. (default: 'storybook')
+        `,
```

Comment:
It'd sure be nice for brevity/DX if it was `npx chromatic init --playwright` instead of `npx chromatic init -f playwright`.  Hopefully people would copy-paste but if not it's one more thing to type. 

I guess conceptually the `-f` is nice since it's really a key-value pair, so take that for what it's worth.

Reply:
Yeah, this one I struggled with because we either are adding a new CLI flag for every framework or an option on a single CLI flag. I could go either way here. 

Moreover, having multiple flags means we need to add extra validation if they pass both —cypress and —playwright. We would also have to do the same for additional frameworks.

Reply:
Good point. @tevanoff @byebrianwong what are your thoughts on this?

Reply:
Good question! I don't feel strongly either way since it's a one-off command. One flag with different options seems fine to me.